instrumentation/net/http/otelhttp/config.go

```diff
@@ -194,3 +197,11 @@ func WithServerName(server string) Option {
 		c.ServerName = server
 	})
 }
+
+// WithDefaultAttributes returns an option that sets the default attributes to be
+// included in metrics.
+func WithDefaultAttributes(attributes []attribute.KeyValue) Option {
```

Comment:
Those attributes will only be used in metrics. Using `DefaultAttributes` as the name could be misleading, as otelhttp also does spans.
How about `WithMetricAttributes`?

Reply:
Sure, will change it